ngraph/frontend/onnx_import/src/op/org.openvinotoolkit/group_norm.cpp

```diff
@@ -0,0 +1,203 @@
+                    std::shared_ptr<ngraph::Node>
+                        create_scale_bias_shape(const Output<ngraph::Node>& node,
+                                                size_t expected_rank_size)
+                    {
+                        const auto& pshape = node.get_partial_shape();
+                        size_t rank_size = pshape.rank().get_length();
+                        if (pshape.is_static())
+                        {
+                            const auto& shape = pshape.to_shape();
+                            std::vector<size_t> new_shape;
+                            if (rank_size == 1)
+                            {
+                                new_shape.push_back(1);
+                                new_shape.push_back(shape[0]);
+                            }
+                            else
+                            {
+                                for (size_t i = 0; i < rank_size; i++)
+                                {
+                                    new_shape.push_back(shape[i]);
+                                }
+                            }
+                            auto i = new_shape.size();
+                            for (; i < expected_rank_size; i++)
+                            {
+                                new_shape.push_back(1);
+                            }
+                            return default_opset::Constant::create(
+                                element::i64, Shape{expected_rank_size}, new_shape);
+                        }
+
+                        auto shape = std::make_shared<default_opset::ShapeOf>(node);
+                        auto splits = builder::opset1::split(shape, rank_size);
+                        auto one = default_opset::Constant::create(element::i64, Shape{1}, {1});
+                        NodeVector dims;
+                        if (rank_size == 1)
+                        {
+                            dims.push_back(one);
+                            dims.push_back(splits[0].get_node_shared_ptr());
+                        }
+                        else
+                        {
+                            for (size_t i = 0; i < rank_size; i++)
+                            {
+                                dims.push_back(splits[i].get_node_shared_ptr());
+                            }
+                        }
+                        auto i = dims.size();
+                        for (; i < expected_rank_size; i++)
+                        {
+                            dims.push_back(one);
+                        }
+                        return std::make_shared<default_opset::Concat>(dims, 0);
+                    }
+                }
+            } // detail
+
+            namespace set_1
+            {
+                OutputVector group_norm(const Node& node)
+                {
+                    auto inputs = node.get_ng_inputs();
+                    NGRAPH_CHECK(inputs.size() == 3, "Invalid number of inputs");
```

Comment:
Please add the number of provided and expected number of inputs in the description.